src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -1464,9 +1509,9 @@ private void handleListParts(HttpServletRequest request,
         }
     }
 
-    private void handleUploadPart(HttpServletRequest request,
-            HttpServletResponse response, String containerName,
-            String blobName, String uploadId)
+    private void handleUploadPart(BlobStore blobStore,
+            HttpServletRequest request, HttpServletResponse response,
+            String containerName, String blobName, String uploadId)
```

Comment:
Can you reorder these parameters?  `blobStore` logically belongs next to `containerName`.